app/learning/p_hosts.py

```diff
@@ -0,0 +1,17 @@
+import re
+
+from app.objects.secondclass.c_fact import Fact
+
+
+class Parser:
+
+    def __init__(self):
+        self.trait = 'multiple'  # remote.host.name, remote.host.fqdn
+
+    def parse(self, blob):
+        for p in re.findall(r'dnshostname[\s:]*(.*?)[\r\n]', blob):
+            yield Fact(trait='remote.host.fqdn', value=p)
+        for p in re.findall(r'cn[\s:]*(.*?)[\r\n]', blob):
+            yield Fact(trait='remote.host.name', value=p)
+        for p in re.findall(r'\\\\(.*?)\\', blob):
```

Comment:
this looks pretty broad of a regex... will this only catch host names? or will it catch mostly anything with a period, such as an email address, a file path with a period in it, an IP address, etc.

can you add a test covering this parser?

Reply:
yea, I can add a test and its actually matching on content between double backslashes and a slash, eg:
`\\comp_hostname\`
so it would grab everything between the backslashes and present that as a match based on the non-greedy match between the parentheses.

Reply:
yeah, my preference on this one would be:

1) add a test
2) make this output generically a host.fqdn.address (or something generic)

I think an FQDN fact is likely something that can be generic. maybe we veer away from host. and use "global." or "domain." to show that..? my point being, this type of fact is likely generic